recipes/dpctl/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This is necessary to help DPC++ find Intel libraries such as SVML, IRNG, etc in build prefix
+export LIBRARY_PATH="$LIBRARY_PATH:${BUILD_PREFIX}/lib"
+
+# Intel LLVM must cooperate with compiler and sysroot from conda
+echo "--gcc-toolchain=${BUILD_PREFIX} --sysroot=${BUILD_PREFIX}/${HOST}/sysroot -target ${HOST}" > icpx_for_conda.cfg
```

Comment:
This should be in dpcpp_linux-64 package. Please open an issue in intel-compiler-repack-feedstock.